### DISCUSSION TITLE
Video of setting up local saleor 3.0 (also paid request)

### DISCUSSION THREAD

**Discussion by D1ll1ng** [saleor/saleor#7723] [General] [answered]
### Problem
I'm really having trouble setting up Saleor both locally and for production, there is some information im missing no matter how much time I put into it there is always new errors in the horizon.

### Request
A video of someone setting up Saleor first and foremost locally, but also for production.

### Alternative request
If you feel you have a pretty good grasp of setting up Saleor, then I would love to talk to you and have you helping me setting it up, I don't mind paying for this information, so I say if you can help me setting it up for me over a video call I will give you 200 euro, and recreate what I have learned in a video for others in the community to use.

**Comment by karolkielecki** [comment, marked as answer]
@D1ll1ng Hi, Please share some screenshots of these errors. You can find our [official docs here](https://docs.getsaleor.com/docs/). If you are looking for videos, you can find some unofficial Youtube video tutorials on setting up Saleor.

- https://www.youtube.com/watch?v=r042HAv-mBI&list=PLzt7OcHd8ypH9mq6bwttlON3PiUzB-GJl

- https://www.youtube.com/watch?v=8YQxJZAYHBA

Cheers and good luck!
